src/Defaults/MergeTag/Taxonomy/TaxonomyName.php

```diff
@@ -32,22 +24,18 @@ class TaxonomyName extends StringTag {
 	 */
 	public function __construct( $params = [] ) {
 
-		if ( isset( $params['taxonomy'] ) ) {
-			$this->taxonomy = $params['taxonomy'];
-		} else {
-			$this->taxonomy = 'category';
-		}
+		$this->set_trigger_prop( $params['taxonomy'] ?? 'taxonomy' );
```

Comment:
```suggestion
		$this->set_trigger_prop( $params['taxonomy'] ?? 'category' );
```

This one's different. Would you mind checking why? Ideally, I'd make it unified just like you did, but check which trigger uses it this way

Reply:
<img width="524" alt="Screenshot 2022-10-14 at 09 17 53" src="https://user-images.githubusercontent.com/70862071/195786087-89a7ab16-61ab-418c-a666-56ac2eb243c0.png">
I think it should stay as 'taxonomy' when $param is null